requirements.txt

```diff
@@ -1 +1,3 @@
-
+beautifulsoup4
+requests
+lxml
```

Comment:
it is a good practice to 
1) specify version as well
2) name required modules in an alphabetical order